crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -28,7 +28,12 @@ pub enum EcmascriptInputTransform {
     ClientDirective(StringVc),
     CommonJs,
     Custom(CustomTransformVc),
-    Emotion,
+    Emotion {
+        #[serde(default)]
+        sourcemap: bool,
```

Comment:
Can we just always enable the sourcemap?

Reply:
for now, this inherits what next.config.js allow to configure. I hope to simplify / consolidate but thinking about feature parity to existing next.js configs we may need to expose.